package.json

```diff
@@ -26,14 +26,15 @@
     "clsx": "^2.1.0",
     "geist": "^1.2.2",
     "immer": "^10.1.1",
-    "lucide-react": "^0.312.0",
+    "lucide-react": "^0.447.0",
     "next": "^14.1.1",
     "node-appwrite": "^13.0.0",
     "postcss": "8.4.29",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.51.3",
     "react-hot-toast": "^2.4.1",
+    "react-icons": "^5.3.0",
```

Comment:
Why is this being updated?

Reply:
In an effort to try and create continuity with our website, I was trying to use lucide-react icons for the social media icons in the team-members section.

They are deprecating a lot of brand icons, including GitHub. Their suggested alternative was doing the same thing.

I found a solution that is not deprecating brand icons.

Reply:
Make sense. We can keep both for now.